hawc/apps/animal/templates/animal/endpoint_detail.html

```diff
@@ -14,8 +14,8 @@
           <div class="dropdown-divider"></div>
           <span class="dropdown-header">BMD Modeling</span>
           {% if bmd_session %}
-            <a class="dropdown-item" href="{{bmd_session.get_absolute_url}}">View session</a>
-            <a class="dropdown-item" href="{% url 'bmd:session_list' object.pk %}">Other options</a>
+            <a class="dropdown-item" href="{{bmd_session.get_absolute_url}}">Selected session</a>
+            <a class="dropdown-item" href="{% url 'bmd:session_list' object.pk %}">All sessions</a>
```

Comment:
These dropdown items appear again in the template, so those should be updated too; this small duplication was to fix #500

Reply:
good find!